catalogue_graph/terraform/locals.tf

```diff
@@ -19,7 +19,7 @@ locals {
 
   # This is a hint that the ingestors might need to be in the pipeline stack!
```

Comment:
Not for addressing in this PR here, but I wonder if the above comment is becoming more true with the changes were making here.